src/core/vpn.tf

```diff
@@ -71,3 +71,45 @@ module "dns_forwarder" {
 
   tags = var.tags
 }
+
+# DNS FORWARDER FOR DISASTER RECOVERY
+
+#
+# DNS Forwarder
+#
+module "dns_forwarder_pair_subnet" {
+  source                                         = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v2.0.3"
```

Comment:
Please use the last version of the module

Reply:
done